### PR TITLE
Headings font style

### PR DESCRIPTION
**Pull request by MartijnCuppens**
Optional variable to set the font style of the headings.

### CODE REVIEW

**Comment by XhmikosR** [comment]
I definitely like it 👍 

But let's wait for one more approval.

**Comment by MartijnCuppens** [comment]
Ping @twbs/css-review

**Review by mdo — APPROVED** [review]
Love that `null` life :D.

**Pull request merged**
